nautobot/users/tests/test_views.py

```diff
@@ -262,7 +262,8 @@ def test_clear_saved_view(self):
             "table_config": {"LocationTable": {"columns": ["name", "status", "location_type", "tags"]}},
         }
         instance.validated_save()
-        self.add_permissions("users.change_savedview")
+        # Custom action requires custom permission
+        self.add_permissions("users.clear_savedview")
```

Comment:
That's definitely not ideal... will be a tripping point for users setting up permissions.